html/static/script.js

```diff
@@ -67,13 +67,52 @@ function saveNotes(element, uid) {
   })
 }
 
+function clearNotesMissing(element, dataset_name, campaign) {
+  $(element).parent().find('textarea').get(0).value = '';
+  let data = {"dataset_name": dataset_name, "campaign": campaign, "missing_nts": notes};
```

Comment:
`notes` here will be undefined which will probably be `None` in python. Just use empty string instead of notes here.

Or you can even delete this line after you read my comment for the next line.